funlet-simple-message/.env

```diff
@@ -1,3 +1,24 @@
+# description: The first URL to <Play>, or string to <Say>.
+# format: text
+# required: true
+FUNLET_MESSAGE1=Hello, world!
```

Comment:
also think FUNLET_MESSAGE_1 would look better on the CE side of things since the space would be introduced

Reply:
Good idea! I changed these to add the space.